backend/mora/service/manager.py

```diff
@@ -13,20 +13,145 @@
 
 """
 import uuid
+import flask
 
 from . import address
 from . import handlers
+from . import orgunit
+from . import facet
+from . import employee
 from .. import common
 from .. import lora
 from .. import mapping
 from .. import util
 from .. import validator
 
+SEARCH_FIELDS = {
+    'e': 'tilknyttedebrugere',
```

Comment:
Duplicates the same field in `address.py`, so perhaps move it to `handlers.py`?

Reply:
Det kan der være noget om, men er det generelt nok? Det er jo foreløbigt blot 2, der anvender det. Jeg plejer først at rykke ved 4.